content/millennium/r4/financial/coverage.md

```diff
@@ -89,6 +93,11 @@ Search for Patient-level or Encounter-level Coverages that meet supplied query p
 <%= headers status: 200 %>
 <%= json(:r4_coverage_encounter_bundle) %>
 
+#### Response when public healthcare information is returned
```

Comment:
let's put this in its own example.

Example - Encounter-level Public Healthcare Coverage

Reply:
added example [here](https://github.com/cerner/fhir.cerner.com/pull/469/commits/60a49b0cd9a1736f87d9db02aec450bd377466b3)